pkg/apis/maistra/conversion/security.go

```diff
@@ -158,16 +158,7 @@ func populateSecurityValues(in *v2.ControlPlaneSpec, values map[string]interface
 			extraArgs := []string{
 				"--tlsCertFile=/etc/cert-manager/tls/tls.crt",
 				"--tlsKeyFile=/etc/cert-manager/tls/tls.key",
-				"--caCertFile=/etc/cert-manager/tls/ca.crt",
-			}
-
-			suppliedRootConfigMap := (CertManagerConf.RootCAConfigMapName != "")
-
-			if suppliedRootConfigMap {
-				extraArgs[2] = "--caCertFile=/etc/cert-manager/ca/root-cert.pem"
```

Comment:
To be honest, I don't understand why it was suggested to override `--caCertFile`, because secret `istiod-tls/ca.crt` and config map `istio-ca-root-cert/root-cert.pem` contains the same certificate.
@rcernich @SpectralHiss any idea?

Reply:
It is safer to use the supplied configmap , in cert-manager it is not guaranteed that in the future that ca.crt key will exist , and it will be retired in cert-manager 1.11 , the root is managed by istio-csr in this installation, i don't think we should change that, as mentioned in the email  , we won't need this change once we release an updated version of istio-csr that will write the root to istio-ca-secret.

Reply:
but if you think it makes things much simpler i wouldn't disagree too hard.

Reply:
No, I don't think so. Thank you for explaining this. I will change this to use the config map.